spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -16,10 +16,213 @@ public: True
 include:
   - types.yaml
   - gnss_signal.yaml
+  - observation.yaml
```

Comment:
Hmmm... a little worried about this pattern of modules taking dependencies on other modules - we haven't done that before. Can we lift out GPSTime into its own module? As we're going towards multi constellation, is GPSTime something we want to generalize (that's likely orthogonal to this issue)?

I guess I'd like to see a common `gnss` module where common things like this would go into - gnss_signal, gps_time. Pulling in other modules internal structures feels like a bad pattern to start. There's probably some other internal structures to do similar with (latency, etc.).

/cc @ljbade @denniszollo @mookerji